test/e2e/harness/protractor.conf.js

```diff
@@ -14,6 +14,9 @@ const SpecReporter = require('jasmine-spec-reporter').SpecReporter;
 const JUnitXmlReporter = require('jasmine-reporters').JUnitXmlReporter;
 const TEST_RESULT_FILE_DIR = '../../../test-reports/e2e';
 
+// eslint-disable-next-line node/no-unpublished-require
+require('../../../env'); // will load environment vars from testenv file and set on process.env
```

Comment:
will E2E no longer respect the testenv file?

Reply:
It failed to bind the testenv when I tried `yarn test:e2e` and `yarn protractor` in the test folder.

Reply:
maybe the testenv file is malformed or in the wrong location. message me and we can troubleshoot.

Reply:
Only the environment variables used by the specs would be affected here: https://github.com/okta/okta-react/blob/master/test/e2e/harness/e2e/App.test.js#L40 USERNAME, PASSWORD

The harness app reads them when it webpacks: https://github.com/okta/okta-react/blob/master/test/e2e/harness/config-overrides.js#L4 ISSUER, CLIENT_ID

Reply:
right, I think the protractor script should support locally e2e execution properly. With this line, it can wire user credentials from testenv. 

Any concern for adding this line?

Reply:
No concern, this is the right place to modify environment vars for protractor